Card.css

```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: "comic sans ms";
+  display: flex;
+  gap : 1.25rem;
+  flex-wrap: wrap;
+  
+}
+
+.card-container {position: relative;
+  display : flex;
+  
+  /* flex-wrap : wrap; */
+  flex-direction : column;
+  justify-content: space-between;
+  background-color: lightgray;
+  border : 0.125rem solid black;
```

Comment:
Don't use fixed colors, save the color in some variable and then use them across your library.